integration-tests/deployment/ccip/test_helpers.go

```diff
@@ -232,3 +248,68 @@ func AddLanesForAll(e deployment.Environment, state CCIPOnChainState) error {
 	}
 	return nil
 }
+
+const (
+	// nolint:lll
+	// https://github.com/smartcontractkit/chainlink/blob/a348b98e90527520049c580000a86fb8ceff7fa7/contracts/src/v0.8/tests/MockV3Aggregator.sol#L76-L76
+	MockLinkAggregatorDescription = "v0.8/tests/MockV3Aggregator.sol"
+	// MockWETHAggregatorDescription WETH use description from MockETHUSDAggregator.sol
+	// nolint:lll
+	// https://github.com/smartcontractkit/chainlink/blob/a348b98e90527520049c580000a86fb8ceff7fa7/contracts/src/v0.8/automation/testhelpers/MockETHUSDAggregator.sol#L19-L19
+	MockWETHAggregatorDescription = "MockETHUSDAggregator"
+)
+
+var (
+	MockLinkPrice = big.NewInt(5e18)
+	// MockDescriptionToTokenDescriptor maps a mock feed description to token descriptor
+	MockDescriptionToTokenDescriptor = map[string]TokenDescriptor{
+		MockLinkAggregatorDescription: LinkDescriptor,
+		MockWETHAggregatorDescription: WETHDescriptor,
+	}
+)
+
+func DeployFeeds(lggr logger.Logger, chain deployment.Chain) (deployment.AddressBook, map[string]common.Address, error) {
+	ab := deployment.NewMemoryAddressBook()
+	linkTV := deployment.NewTypeAndVersion(PriceFeed, deployment.Version1_0_0)
+	mockLinkFeed, err := deployContract(lggr, chain, ab,
+		func(chain deployment.Chain) ContractDeploy[*aggregator_v3_interface.AggregatorV3Interface] {
+			linkFeed, tx, _, err1 := mock_v3_aggregator_contract.DeployMockV3Aggregator(
+				chain.DeployerKey,
+				chain.Client,
+				DECIMALS,      // decimals
+				MockLinkPrice, // initialAnswer
+			)
+			aggregatorCr, err2 := aggregator_v3_interface.NewAggregatorV3Interface(linkFeed, chain.Client)
+
+			var err error
+			if err1 != nil || err2 != nil {
+				err = fmt.Errorf("linkFeedError: %w, AggregatorInterfaceError: %w", err1, err2)
+			}
+			return ContractDeploy[*aggregator_v3_interface.AggregatorV3Interface]{
+				Address: linkFeed, Contract: aggregatorCr, Tv: linkTV, Tx: tx, Err: err,
+			}
+		})
+
+	if err != nil {
+		lggr.Errorw("Failed to deploy link feed", "err", err)
+		return ab, nil, err
+	}
+
+	lggr.Infow("deployed mockLinkFeed", "addr", mockLinkFeed.Address)
+
+	desc, err := mockLinkFeed.Contract.Description(&bind.CallOpts{})
+	if err != nil {
+		lggr.Errorw("Failed to get description", "err", err)
+		return ab, nil, err
+	}
+
+	if desc != MockLinkAggregatorDescription {
```

Comment:
Having a proper description would make this assertion more useful, cuz right now the description of a mock link feed and a mock weth feed will be the same right?